### ISSUE TITLE
Unknown timezone warning

### ISSUE THREAD

**Issue by xiangzhu** [workflowr/workflowr#69]
Hi John,

I updated an iOS machine to macOS High Sierra 10.13 yesterday, and got the following warning sign at the top of each page:

```r
Warning in as.POSIXlt.POSIXct(Sys.time()): unknown timezone 'default/
America/Los_Angeles'
``` 

Here is a full example with session information: https://xiangzhu.github.io/stanford-stats110/count_prob

Thanks in advance for your help!

**Comment by pcarbo** [comment]
@xiangzhu I believe this is a MacOS-specific issue that has nothing to do with workflowr. Which version of R are you using? Some suggestions: (1) Try setting the time zone in System Preferences; (2) Try upgrading R to the latest version. For more technical details (and possible solutions) see [here](https://stackoverflow.com/questions/4047188/unknown-timezone-name-in-r-strptime-as-posixct).

**Comment by xiangzhu** [comment]
@pcarbo  Hi Peter,

Thanks for your reply!

1. Here is my time zone setting.

<img width="608" alt="screen shot 2017-09-26 at 6 46 16 pm" src="https://user-images.githubusercontent.com/5986023/30892005-146ddcde-a2eb-11e7-99f4-c21b573d647f.png">

2. I am using R version 3.4.1.

```r
> sessionInfo()
R version 3.4.1 (2017-06-30)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.4.1 tools_3.4.1 
```

Also thanks for sharing the stackoverflow thread. I saw it before I opened this ticket. I could not get a solution from the discussions there.

**Comment by xiangzhu** [comment]
@jdblischak  @pcarbo 
Currently I have to manually add `warning=FALSE` to the following auto-generated chunk:

```rmd
<!-- Insert the date the file was last updated -->
```{r last-updated, echo=FALSE, results='asis', warning=FALSE}
```

I wish there is a solution that I do not have to modify chunks generated by `workflowr`. Thanks!

**Comment by pcarbo** [comment]
@xiangzhu I'm sorry, I can't replicate your bug. It sounds like it is an issue that was caused by upgrading to MacOS Sierra. Did you re-install R after upgrading? Mac sometimes moves system files around when they introduce a new OS. (The Stackoverflow post was mostly about R on Linux.)

**Comment by xiangzhu** [comment]
@pcarbo  If I re-install R, should I also re-install all `R` packages? If so, I prefer to manually add `warning=FALSE` at this moment.

**Comment by pcarbo** [comment]
@xiangzhu I believe it should keep all your packages. To be safe, I would make a copy of the directories listed in `.libPaths()` before reinstalling.

**Comment by xiangzhu** [comment]
@pcarbo  Thanks for your quick reply! I will wait and try this when I become more familiar with iOS :)

**Comment by jdblischak** [comment]
@xiangzhu Sorry to hear the macOS upgrade disrupted your R installation. I agree with Peter that re-installing R may be the best option.

@pcarbo Thanks for handling this one.

Since the only possible workflowr-based solution has been found, i.e. to add `warning=FALSE` to the date chunk, I am going to close this Issue. @xiangzhu if you find a more long-term solution, please do post it here for others even though the Issue is closed.

**Comment by jdblischak** [comment]
This warning should hopefully be fixed now with the release of [R 3.4.3](https://stat.ethz.ch/pipermail/r-help/2017-November/450302.html):

```
    * A workaround has been added for the changes in location of
      time-zone files in macOS 10.13 'High Sierra' and again in
      10.13.1, so the default time zone is deduced correctly from the
      system setting when R is configured with --with-internal-tzcode
      (the default on macOS).
```
